src/cephadm/cephadm

```diff
@@ -2844,7 +2844,14 @@ def command_bootstrap():
                               '--allow-overwrite to overwrite' % f)
         dirname = os.path.dirname(f)
         if dirname and not os.path.exists(dirname):
-            raise Error('%s directory %s does not exist' % (f, dirname))
+            fname = os.path.basename(f)
+            logger.info(f"Creating directory {dirname} for {fname}")
+            try:
+                # use makedirs to create intermediate missing dirs
+                os.makedirs(dirname, 0o700)
```

Comment:
This might want to be 0o755 (at least, that's the permissions I'd expect /etc/ceph to have)

Reply:
ACK - updated